sdk/communication/Azure.Communication.Chat/CHANGELOG.md

```diff
@@ -1,13 +1,21 @@
 # Release History
 
-## 1.0.0-beta.4 (2021-02-08)
-# Breaking
-- Support for CreateChatThreadResult and AddChatParticipantsResult to handle partial errors in batch calls.
+## 1.0.0-beta.4 (2021-02-09)
+
+### Breaking Changes
+
+- Replaced `CommunicationUser` with `CommunicationUserIdentifier`.
```

Comment:
We used to say "Renamed" (as far as I remember). Should we always use the same (either Replaced... with or Renamed ... to)? Just to be perfectly professional. :-)

Reply:
Changed to Renamed.

Reply:
Wait a minute. This is Chat ChangeLog.md. Why are we talking about CommunicationUserIdentifier here? Shouldn't that belong to Common ChangeLog.md?